doc/classes/PhysicsBody2D.xml

```diff
@@ -25,12 +25,12 @@
 		</method>
 		<method name="move_and_collide">
 			<return type="KinematicCollision2D" />
-			<argument index="0" name="linear_velocity" type="Vector2" />
+			<argument index="0" name="distance" type="Vector2" />
 			<argument index="1" name="test_only" type="bool" default="false" />
 			<argument index="2" name="safe_margin" type="float" default="0.08" />
 			<description>
-				Moves the body along the vector [code]linear_velocity[/code]. This method should be used in [method Node._physics_process] (or in a method called by [method Node._physics_process]), as it uses the physics step's [code]delta[/code] value automatically in calculations. Otherwise, the simulation will run at an incorrect speed.
-				The body will stop if it collides. Returns a [KinematicCollision2D], which contains information about the collision when stopped, or when touching another body along the motion.
+				Moves the body along the vector [code]distance[/code]. In order to be frame rate independent in `_physics_process` or `_process`, [code]distance[/code] should be computed using [code]delta[/code].
```

Comment:
```suggestion
				Moves the body along the vector [code]distance[/code]. In order to be frame rate independent in [method Node._physics_process] or [method Node._process], [code]distance[/code] should be computed using [code]delta[/code].
```

Reply:
Same for the other occurrences. For the reference, <code>`foo`</code> doesn't work for inline code formatting in the XML markup, it should be <code>[code]foo[/code]</code>. But here it's better to refer to the actual methods with a hyperlink.